FirebaseInstanceID.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'FirebaseInstanceID'
-  s.version          = '4.1.0'
+  s.version          = '4.1.2'
```

Comment:
These InstanceID podspecs are going to cause SpecsStaging conflicts with #2880. Ultimately, the next release should be 4.1.0 and in the meantime perhaps a merged podspec from both PR's should be on SpecsStaging

cc: @chliangGoogle

Reply:
@paulb777 Thank you for noticing this. I think, we will need to make adjustments to the versions before merging this PR anyways. Would it make sense to keep the current podspec independent and update it right before merging?

Reply:
@maksymmalyhin Unfortunately, until local pod spec linting is supported in CocoaPods 1.7.0, we need to push InstanceID to SpecsStaging for the FirebaseMessging `pod spec lint` to pass on travis - if Messaging depends on changed public or private IID APIs.

Reply:
I am going to put this PR on hold so far until we make a final decision if we have to release this change at all. So, I think, @chliangGoogle can safely proceed with her PR ignoring this so far.

If we need to release this change, we will revisit pods versions.